cdl/export_route_service.go

```diff
@@ -0,0 +1,190 @@
+package cdl
+
+import (
+	"fmt"
+	"github.com/go-playground/validator/v10"
+	"io"
+	"net/http"
+	"path"
+)
+
+type ExportRoute struct {
+	ID              string               `json:"id,omitempty"`
+	ExportRouteName string               `json:"name" validate:"required"`
+	Description     string               `json:"description,omitempty"`
+	DisplayName     string               `json:"displayName" validate:"required"`
+	Source          Source               `json:"source" validate:"required"`
+	AutoExport      bool                 `json:"autoExport,omitempty"`
+	Destination     Destination          `json:"destination" validate:"required"`
+	ServiceAccount  ExportServiceAccount `json:"serviceAccount" validate:"required"`
+	CreatedBy       string               `json:"createdBy,omitempty"`
+	CreatedOn       string               `json:"createdOn,omitempty"`
+	UpdatedBy       string               `json:"updatedBy,omitempty"`
+	UpdatedOn       string               `json:"updatedOn,omitempty"`
+}
+
+type Source struct {
+	CdlResearchStudy ExportResearchStudySource `json:"cdlResearchStudy" validate:"required"`
```

Comment:
Please rename to `CDLResearchStudy` as per `Go` convention

Reply:
Changed CdlResearchStudy -> CDLResearchStudy